enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -186,6 +198,10 @@ func TestWithPublicRepo(t *testing.T) {
 	})
 	require.NoError(t, err)
 	require.Contains(t, string(logResp.GetBuffer()), "Usage: bazel <command> <options>")
+
+	t.Cleanup(func() {
+		resetFlags(t)
+	})
```

Comment:
I think this cleanup should go at the top of each test so that it runs even if the `require` calls fail above?

Reply:
Ack - fixed